recipes-kernel/linux/linux-yocto-dev.bbappend

```diff
@@ -31,3 +31,8 @@ KBUILD_DEFCONFIG:qcom ?= "defconfig"
 do_install:append:qcom() {
 	sed -i 's:${TMPDIR}::g' ${WORKDIR}/linux-${PACKAGE_ARCH}-${LINUX_KERNEL_TYPE}-build/drivers/gpu/drm/msm/generated/*
 }
+
+# For dtb.bin
+QCOM_DTBBIN = ""
+QCOM_DTBBIN:qcom = "linux-qcom-dtbbin"
+inherit ${QCOM_DTBBIN}
```

Comment:
Because this `inherit` is somehow conditional we must use the `inherit_defer` instead.
I agree with the suggestion to use the KERNEL_CLASSES for that who already uses `inherit_defer`.